shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -176,15 +174,11 @@ - (instancetype)initWithEngine:(FlutterEngine*)engine
   _terminator = terminator ? terminator : ^(id sender) {
     // Default to actually terminating the application. The terminator exists to
     // allow tests to override it so that an actual exit doesn't occur.
-    FlutterApplication* flutterApp = [FlutterApplication sharedApplication];
-    if (flutterApp && [flutterApp respondsToSelector:@selector(terminateApplication:)]) {
-      [[FlutterApplication sharedApplication] terminateApplication:sender];
-    } else if (flutterApp) {
-      [flutterApp terminate:sender];
-    }
+    NSApplication* flutterApp = [NSApplication sharedApplication];
```

Comment:
Is there any difference between `[NSApplication sharedApplication]` and `NSApp`?  Is one better than the other style-wise?

Reply:
There's no difference; NSApp is just a global variable containing `[NSApplication sharedApplication]`. Not sure if there's ever a style reason to prefer one over the other. @stuartmorgan might know.